packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -1342,6 +1376,38 @@ void main() {
     expect(exception, isFlutterError);
     expect(exception.toString(), contains('Every item of ReorderableListView must have a key.'));
   });
+
```

Comment:
Should we also include a test where the Overlay does not tightly wrap the list (i.e. the list has to share the screen space with another sibling) to ensure that our math is correct to the origin of the overlay and not the origin of the list?